portality/models/background.py

```diff
@@ -84,6 +84,7 @@ def add_audit_message(self, msg, timestamp=None):
             timestamp = dates.now_with_microseconds()
         obj = {"message": msg, "timestamp": timestamp}
         self._add_to_list_with_struct("audit", obj)
+        print(msg)
```

Comment:
do we want print statements rather than logging?

Reply:
no, we're logging them as we used to and I just added the print statement at the end of log to make test debugging easier.